packaging/services/ovirt-engine-notifier/ovirt-engine-notifier.conf.in

```diff
@@ -17,7 +17,7 @@ PACKAGE_DISPLAY_VERSION="@DISPLAY_VERSION@"
 # Comma separated list of keys
 # not to be logged.
 #
-SENSITIVE_KEYS=""
+SENSITIVE_KEYS="KEYCLOAK_DB_PASSWORD,EXTERNAL_OIDC_CLIENT_SECRET,EXTERNAL_OIDC_HTTPS_PKI_TRUST_STORE_PASSWORD"
```

Comment:
Functionality is fine. For style, perhaps keep this empty and append later like with other keys later in the file.

Reply:
No fully understand why this offends a style. The only other place where something is appended to SENSITIVE_KEYS is a place where mail related properties are defined which actually makes sense to keep them together. In case of OIDC/keycloak parameters these are not defined here so they do not own .  Perhaps they should be?

However, ovirt-engine.conf.in is another story - I will  fix it there.

Reply:
Actuall after fixing that for engine-conf  I changed my mind because it actually looks better :) Fixed in notifier as well.  Thanks Didi!